app/views/resources/index.html.erb

```diff
@@ -14,6 +14,21 @@
         <% end %>
       </ul>
     </div>
+
+    <div class="dropdown">
+      <button class="btn btn-secondary dropdown-toggle" type="button" id="allResourcesButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        All Resources
+      </button>
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+      <% @categories.each do |category| %>
+        <li data-category="<%= create_filter(category.id) %>">
+          <%= link_to category.name, resources_path(anchor: category.id) %>
+        </li>
```

Comment:
`<li></li>` element can not exist without wrapping `<ul></ul>` or `<ol></ol>`. Let's remove `<li>` here and replace it with simple links? per examples for dropdowns on https://getbootstrap.com/docs/4.1/components/dropdowns/

Reply:
thanks for the advice !